code/hud/hudsquadmsg.h

```diff
@@ -56,6 +56,8 @@ class object;
 // out of order, but it was this way in the original source
 #define DISABLE_SUBSYSTEM_ITEM		(1<<15)
 
+#define MAX_ENGINE_PLAYER_ORDER		DISABLE_SUBSYSTEM_ITEM
```

Comment:
Is this setting up a situation where it will no longer be possible to add any items beyond this one?  If so, then it would be better to count backwards from 31 while the built-in orders count upwards from 15.

Reply:
Yes it is such a situation. But we still need this kind of define to warn the modder as to when more orders cannot be added due to collisions with engine orders. What we can get rid of by counting backwards however, is the ``constexpr int max_highest_player_order_type = (1 << 31);``. 
Changed.